### ISSUE TITLE
NVDA will be crashed when microsoft bing input method is installed and run in the computer in windows 8.1 above.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4959]
**Reported by vgjh2005 on 2015-03-05 14:08**
Hi：
NVDA cannot support TSF. So bing is often used in Windows 8.1 above. Because bing is made by two frames: tsf and IME. In normal wwindow, IME is used to make input. In Windows 8.1 apps windows or start screen, TSF is used. But there is a credical BUG on it.
1. Please enter NVDA menu-> options-> general settings and select "Show exiting options when exiting NVDA".
2. Install bing input method in your computer.
3. Press windows+spacebar to toggle to Microsoft Bing input method.
4. Press NVDA+Q to try to exit NVDA. Now, computer will be crashed. We cannot do anything unless restart your computer.
5. If this event isn't occured, please set nvda to start automatically then restart your computer. NVDA will not start at this time.
Hopeful to support TSF quickly. Thanks!

**Comment by bhavyashah** [comment]
@vgjh2005  Do you still experience the reported crash with NVDA 2017.3? Also, @jcsteh, do you agree that the TSF support issues highlighted in https://github.com/nvaccess/nvda/issues/4959#issuecomment-155334873 should be covered by a separate ticket, or do you believe a combined approach to both the crash and the TSF support problems is more logical?

**Comment by jcsteh** [comment]
I don't think we have sufficient info to split this into two issues yet.

**Comment by ehollig** [comment]
Closing as requested in https://github.com/nvaccess/nvda/issues/4959#issuecomment-328396773
